### DISCUSSION TITLE
LOVE IT !!!

### DISCUSSION THREAD

**Discussion by opensourcefan** [mattieserver/netbox-topology-views#315] [General]
Thank you for this very cool plugin. I'm enjoying it very much. I'm not much of a coder but I can help with ideas if need be. 

Install went fine once I figured it all out. Important to remember if you're using `sudo` while in venv you should use `sudo -E` or it will execute outside of the venv.

**Comment by dreng** [comment]
Thank you for the warm words! Of course, any contribution is appreciated. That's what makes open source so sexy.
